mumlib/src/command.rs

```diff
@@ -67,9 +67,14 @@ pub enum CommandResponse {
     },
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum ChannelTarget {
```

Comment:
Document this struct.

Reply:
Also, this enum is the same as `Option`. What is the reasoning behind making it its own thing?

Reply:
Because I think `MessageTarget::Channel(vec![(None, false)])` is misleading without reason compared to `MessageTarget::Channel(vec![(ChannelTarget::Default, false)])`. I feel like None => default channel is too implicit.

Reply:
Yeah sure seem reasonable.